nnpdf_data/nnpdf_data/theory_cards/41000011.yaml

```diff
@@ -0,0 +1,43 @@
+ID: 41_000_011
+Comments: NNPDF4.1 NNLO QCD-only (xir, xif) = (1, 0.5)
+PTO: 2
+FNS: FONLL-C
+DAMP: 0
+IC: 1
+Q0: 1.65
+ModEv: EXA
+ModSV: expanded
```

Comment:
Dropping `ModSV` is a solution (as we found out), but if you do so, please do it consistently (as now it is halfway and confusing, I'd say)

Reply:
From the discussion I thought it should be not defined or None _only_ for XIF==1.

Reply:
I confused myself, sorry, you're right